src/util/uuid.h

```diff
@@ -6,7 +6,5 @@
 
 #pragma once
 
-#include "../../graphcontext.h"
-#include "../../../redismodule.h"
-
-GraphContext *RdbLoadGraphContext(RedisModuleIO *rdb);
+// Generates a new UUID.
+char *UUID_New();
```

Comment:
What happened here? why is this file renamed?

Reply:
don't know?